daprdocs/content/en/reference/resource-specs/subscription-schema.md

```diff
@@ -0,0 +1,85 @@
+---
+type: docs
+title: "Subscription spec"
+linkTitle: "Subscription spec"
+weight: 2000
+description: "The basic spec for a Dapr subscription"
+---
+
+The `Subscription` Dapr resource allows you to subscribe declaratively to a topic using an external component YAML file.
+
+## `v1alpha1`
+
+The following is the basic version `v1alpha1` spec for a `Subscription` resource. 
+
+```yml
+apiVersion: dapr.io/v1alpha1
+kind: Subscription
+metadata:
+  name: <REPLACE-WITH-RESOURCE-NAME>
+spec:
+  version: v1alpha1
+  topic: <REPLACE-WITH-TOPIC-NAME> # Required
+  route: <REPLACE-WITH-ROUTE-NAME> # Required
+  pubsubname: <REPLACE-WITH-PUBSUB-NAME> # Required
+  deadLetterTopic: <REPLACE-WITH-DEAD-LETTER-TOPIC-NAME> # Optional
+  bulkSubscribe: # Optional
+  - enabled: <REPLACE-WITH-BOOLEAN-VALUE>
+  - maxmessages: <REPLACE-WITH-VALUE>
+  - maxawaitduration: <REPLACE-WITH-VALUE>
+scopes:
+- <REPLACE-WITH-SCOPED-APPIDS>
+```
+
+### Spec fields
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
+| topic | Y | The name of the topic to which your component subscribes. | `orders` |
+| route | Y | The endpoint to which all topic messages are sent. | `/checkout` |
+| pubsubname | N | The name of your pub/sub component. | `pubsub` |
+| deadlettertopic | N | The name of the dead letter topic that forwards undeliverable messages. | `poisonMessages` |
+| bulksubscribe | N | Enable bulk subscribe properties. | `true`, `false` |
+
+## `v2alpha1`
+
+The following the basic `v2alpha1` spec for a `Subscription` resource.
+
+```yml
+apiVersion: dapr.io/v2alpha1
+kind: Subscription
+metadata:
+  name: <REPLACE-WITH-NAME>
+spec:
+  version: v2alpha1
```

Comment:
I completely forget we introduced a v2 subscription API. We do not mention this anywhere other than here in passing https://docs.dapr.io/developing-applications/building-blocks/pubsub/howto-route-messages/ and it should be the default example. In other words we should consider updating all places that have v1 to v2.

These docs are the ones showing the v1 spec  pubsub-deadletter.md, pubsub-raw.md and subscription-methods.md and should be changed to v2. 

Also we should put v2 one first in the doc and say that the v1 is deprecated. The V2alpha1 spec is the default.